docs(setting): update settings (#1189)
longhorn/website: content/docs/1.10.0/references/settings.md

To configure Longhorn before installing it, see [this section](../../advanced-resources/deploy/customizing-default-settings) for details.

### System Info

#### Default Engine Image

> The default engine image used by the manager. Can be changed on the manager starting command line only.
> 
> Every Longhorn release will ship with a new Longhorn engine image. If the current Longhorn volumes are not using the default engine, a green arrow will show up, indicate this volume needs to be upgraded to use the default engine.

Reviewer: ```suggestion
The default engine image used by the manager. This setting can only be changed at startup via the Longhorn manager's command line arguments. 

Every Longhorn release will be shipped with a new Longhorn engine image. If the current Longhorn volumes are not using the default engine, a green arrow will indicate that a volume needs to be upgraded to use the current default engine.
```

#### Default Instance Manager Image

> The default instance manager image used by the manager. Can be changed on the manager starting command line only.

Reviewer: ```suggestion
The default instance manager image used by the manager. This setting can only be changed at startup via the Longhorn manager's command line arguments.
```

#### Default Backing Image Manager Image

> The default backing image manager image used by the manager. Can be changed on the manager starting command line only.

Reviewer: ```suggestion
The default backing image manager image used by the manager. This setting can only be changed at startup via the Longhorn manager's command line arguments.
```

#### Support Bundle Manager Image

Longhorn uses the support bundle manager image to generate the support bundles.

> There will be a default image given during installation and upgrade. You can also change it in the settings.

Reviewer: ```suggestion
A default image is provided during installation and upgrades, but you can also change this setting.
```

An example of the support bundle manager image:
> Default: `longhornio/support-bundle-kit:v0.0.14`

#### Current Longhorn Version

The current Longhorn version.

#### Latest Longhorn Version

> The latest version of Longhorn available. Automatically updated by the Upgrade Checker.
> 
> > Only available if `Upgrade Checker` is enabled.

Reviewer: ```suggestion
The latest version of Longhorn available, automatically updated by the Upgrade Checker. This information is only available if the Upgrade Checker is enabled.
```

#### Stable Longhorn Versions

> The latest stable version of every minor release line. Updated by Upgrade Checker automatically.

Reviewer: ```suggestion
The latest stable version of every minor release line. This information is automatically updated by the Upgrade Checker.
```
